test cases/failing/59 bad option argument/meson_options.txt

```diff
@@ -1 +1 @@
-option('name', type : 'string', vaule : 'foo')
+option('name', type : 'string', value_ : 'foo')
```

Comment:
The goal of these tests is to show that a divergence from the expected token will be flagged.

My suggestion is that it doesn't require using a word not in the dictionary. (One could also use `values` instead if one wants a nearby word in the dictionary without introducing punctuation.)